ddtrace/contrib/pytest/__init__.py

```diff
@@ -0,0 +1,22 @@
+"""
+Enable traced execution of tests using ``pytest`` runner by
+running ``pytest --ddtrace`` or by modifying any configuration
+file read by Pytest (``pytest.ini``, ``setup.cfg``, ...)::
+
+    [pytest]
+    ddtrace = 1
+
```

Comment:
Let's also add the `DD_PYTEST_SERVICE` configuration option 🙂

Reply:
@Kyle-Verhoog done. I have also added the `DD_PYTEST_OPERATION_NAME` config option.